examples/insurances/retrieve_insurance.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'easypost'
+
+EasyPost.api_key = ENV['EASYPOST_TEST_API_KEY']
+
+EasyPost::Insurance.retrieve('ins_...')
+
+puts insurance
```

Comment:
```suggestion
insurance = EasyPost::Insurance.retrieve('ins_...')

puts insurance
```

You forgot assigning it to a variable